settings.gradle.kts

```diff
@@ -1,5 +1,106 @@
+            library("core-client", "org.eclipse.edc", "registration-service-client").versionRef("registrationService")
+            library(
+                "ext-participantVerifier",
+                "org.eclipse.edc",
+                "participant-verifier"
+            ).versionRef("registrationService")
+
+            library(
+                "ext-registrationPolicy-gaiaxMember",
+                "org.eclipse.edc",
+                "registration-policy-gaiax-member"
+            ).versionRef("registrationService")
+
+        }
+        create("edc") {
+            version("edc", "0.0.1-20221113-SNAPSHOT")
+            library("util", "org.eclipse.edc", "util").versionRef("edc")
+            library("boot", "org.eclipse.edc", "boot").versionRef("edc")
+            library("spi-policy-engine", "org.eclipse.edc", "policy-engine-spi").versionRef("edc")
+            library("spi-contract", "org.eclipse.edc", "contract-spi").versionRef("edc")
+            library("spi-federatedCatalog", "org.eclipse.edc", "federated-catalog-spi").versionRef("edc")
+            library("spi-ids", "org.eclipse.edc", "ids-spi").versionRef("edc")
+            library("spi-dataPlane-selector", "org.eclipse.edc", "data-plane-selector-spi").versionRef("edc")
+
+            library("core-connector", "org.eclipse.edc", "connector-core").versionRef("edc")
+            library("core-policy-engine", "org.eclipse.edc", "policy-engine").versionRef("edc")
+            library("core-policy-evaluator", "org.eclipse.edc", "policy-evaluator").versionRef("edc")
+            library("core-controlPlane", "org.eclipse.edc", "control-plane-core").versionRef("edc")
+            library("core-dataPlane-selector", "org.eclipse.edc", "data-plane-selector-core").versionRef("edc")
+            library("core-dataPlane-framework", "org.eclipse.edc", "data-plane-framework").versionRef("edc")
+            library("core-federatedCatalog", "org.eclipse.edc", "federated-catalog-core").versionRef("edc")
+            library("core-micrometer", "org.eclipse.edc", "micrometer-core").versionRef("edc")
+
+            library("ext-identity-did-core", "org.eclipse.edc", "identity-did-core").versionRef("edc")
+            library("ext-identity-did-service", "org.eclipse.edc", "identity-did-service").versionRef("edc")
+            library("ext-identity-did-web", "org.eclipse.edc", "identity-did-web").versionRef("edc")
+            library("ext-http", "org.eclipse.edc", "http").versionRef("edc")
+            library("ext-auth-tokenBased", "org.eclipse.edc", "auth-tokenbased").versionRef("edc")
+            library("ext-api-dataManagement", "org.eclipse.edc", "data-management-api").versionRef("edc")
+            library("ext-micrometer-jetty", "org.eclipse.edc", "jetty-micrometer").versionRef("edc")
+            library("ext-micrometer-jersey", "org.eclipse.edc", "jersey-micrometer").versionRef("edc")
+            library("ext-observability", "org.eclipse.edc", "api-observability").versionRef("edc")
```

Comment:
should probably be `ext-api-observability`, to keep it constant with e.g. the `management` api